### ISSUE TITLE
Questions about DeeplabV2 pretrained weight

### ISSUE THREAD

**Issue by Unrealluver** [qjadud1994/DRS#2]
Greetings,

Thank you for your code and paper, they give us a novel way to process the relationship with most descriptive part and the other object part. 
When I read your code, I find some question, would you answer them for me?
I find that the pretrained link in your deeplabv2 table turn into the download for a checkpoint file with name "final checkpoint.pth", is that truly the pretrained model for deeplabv2 or just the final checkpoint of your deeplabv2 training? Another reason let me think like this is that I find the pretrained path in your config file is "data/models/coco/deeplabv1_resnet101/caffemodel/deeplabv1_resnet101-coco.pth". Is it the same checkpoint file with the "checkpoint.pth"?

**Comment by qjadud1994** [comment]
Thank you for your interest in our work.

1. `checkpoint_final.pth` is our final checkpoint which produces 70.4% miou.
2. According to the [DeepLab-V2 pytorch implementation](https://github.com/kazuto1011/deeplab-pytorch#download-pre-trained-caffemodels) we used, it requires pretrained weight.
3. We share the `deeplabv1_resnet101-coco.pth` this [[link]](https://drive.google.com/file/d/1Wj8Maj9KGQgwtDfvIp8FChsdAIgDvliT/view?usp=sharing)

**Comment by Unrealluver** [comment]
Thank you for your early reply! I have one question about the pretrained weight of  deeplabv2. 
I wonder if most people would use the coco pretrained in the WSSS domain rather than ImageNet classification pretrain?

**Comment by qjadud1994** [comment]
I also had questions about the pretrained weights.

First, If there is no COCO pretrained weight in the original DeepLab-V2 implementation, performance on VOC2012 (full segmentation label training case) cannot be reproduced.

Therefore, we followed the DeepLab-V2 training strategies.

However, For the DeepLab-V1, we didn't require COCO pretrained weight.

**Comment by Unrealluver** [comment]
Thank you for your reply！
